cmd/blocking-issue-creator/main.go

```diff
@@ -22,29 +20,36 @@ import (
 
 type options struct {
 	promotion.FutureOptions
-	username  string
-	tokenPath string
+	github prowflagutil.GitHubOptions
+
+	githubLogin string
+	dryRun      bool
 }
 
 func (o *options) Validate() error {
+	if o.githubLogin == "" {
+		return errors.New("--github-login is required")
+	}
+
 	if err := o.FutureOptions.Validate(); err != nil {
 		return err
 	}
-	if o.username == "" {
-		return errors.New("--username is required")
-	}
-	if o.tokenPath == "" {
-		return errors.New("--token-path is required")
+	if err := o.github.Validate(o.dryRun); err != nil {
+		return err
 	}
 	return nil
 }
 
 func gatherOptions() options {
 	o := options{}
 	fs := flag.NewFlagSet(os.Args[0], flag.ExitOnError)
-	fs.StringVar(&o.username, "username", "", "Username to use when communicating with GitHub.")
-	fs.StringVar(&o.tokenPath, "token-path", "", "Path to token to use when communicating with GitHub.")
-	o.Bind(fs)
+
+	fs.BoolVar(&o.dryRun, "dry-run", true, "Dry run for testing. Uses API tokens but does not mutate.")
```

Comment:
Yea I have a [follow-up](https://github.com/openshift/release/pull/16280) for that

Reply:
Steve's point is that there will be a period of time where the job is broken (between merging the code change and manifest change). The usual way (which we should do by default) to avoid breakages in cases like this is:

1) Add a new option, keep the old option
2) Move the users to use the new options
3) Remove old option